client/my-sites/site-settings/date-time-format/time-format-option.jsx

```diff
@@ -68,7 +69,7 @@ export const TimeFormatOption = ( {
 			</span>
 			<FormSettingExplanation>
 				<ExternalLink
-					href="https://support.wordpress.com/settings/time-settings/"
+					href={ localizeUrl( 'https://wordpress.com/support/settings/time-settings/' ) }
```

Comment:
Doesn't `ExternalLink` already use `localizeUrl`? Is there harm in doubling up?

Reply:
It does, so this is not necessary. Double `localizeUrl()` is not a problem, though, I recently specifically fixed this in #40653.